packages/skeleton/src/lib/utilities/Modal/Modal.svelte

```diff
@@ -101,8 +101,8 @@
 	export let transitionOutParams: TransitionParams<TransitionOut> = { duration: 150, opacity: 0, x: 0, y: 100 };
 
 	// Base Styles
-	const cBackdrop = 'fixed top-0 left-0 right-0 bottom-0 overflow-y-auto';
-	const cTransitionLayer = 'w-full h-fit min-h-full p-4 overflow-y-auto flex justify-center';
+	const cBackdrop = 'fixed top-0 left-0 right-0 bottom-0 overflow-hidden';
```

Comment:
@CoreyT355 @Mahmoud-zino after reviewing Mahmoud's PR here:
https://github.com/skeletonlabs/skeleton/pull/2255

I've noted you're both modifying the `overflow` property here. Given the only other change is the `max-h-screen` setting below. I'd recommend that if we feel this is useful, to replicate it in @Mahmoud-zino PR and we'll favor that and close this PR.

Let me know if that works for both of you. Thanks!

Reply:
@endigo9740 Definitely better to unify the changes down to a single PR.